Python/PythonIntro.md

```diff
@@ -445,9 +445,43 @@ rover.__doc__
 dir(rover)
 # [__dir__, __doc__, ..., 'bark', 'barks', 'is_happy', 'name', 'owner', 'wagging']
 ```
+```
+Python String
+In Python, Strings are arrays of bytes representing Unicode characters. However, Python does not have a character data type, a single character is simply a string with a length of 1. Square brackets can be used to access elements of the string.
 
+```
+```
+Creating a String
+Strings in Python can be created using single quotes or double quotes or even triple quotes.
+```
+`` Python Program for Creation of String 
+ Creating a String  with single Quotes 
+``
+```py
+String1 = 'Welcome to the Geeks World'
+print("String with the use of Single Quotes: ") 
+print(String1) 
+ ``` 
+ Creating a String  with double Quotes 
+```
+String1 = "I'm a Geek"
+print("\nString with the use of Double Quotes: ") 
+print(String1) 
+  ``` Creating a String  with triple Quotes ```
+String1 = '''I'm a Geek and I live in a world of "Geeks"'''
```

Comment:
maybe it's important to mention why you might use these different ways of writing strings, like to avoid escaping the other set of quotes like you've demonstrated here.